factory_linux.go

```diff
@@ -10,7 +10,9 @@ import (
 	"os/exec"
 	"path/filepath"
 	"regexp"
+	"syscall"
 
+	"github.com/docker/docker/pkg/mount"
```

Comment:
Humm, i forgot about this.  Maybe we want to keep it in docker so that we don't have to add this dep.  We can define this same function there since that is the reason why we have these.